src/webac/webapp/WEB-INF/web.xml

```diff
@@ -1,74 +1,73 @@
 <?xml version="1.0" encoding="UTF-8"?>
-
 <web-app xmlns="http://java.sun.com/xml/ns/javaee"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd"
-	version="3.0">
-
-	<display-name>Fedora 4</display-name>
-
-	<context-param>
-		<param-name>contextConfigLocation</param-name>
-		<param-value>WEB-INF/classes/spring/auth-master.xml</param-value>
-	</context-param>
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd"
+    version="3.0">
 
-	<listener>
-		<listener-class>org.springframework.web.context.ContextLoaderListener</listener-class>
-	</listener>
+    <display-name>Fedora Repository</display-name>
 
-  <servlet>
-    <servlet-name>jersey-servlet</servlet-name>
-    <servlet-class>org.glassfish.jersey.servlet.ServletContainer</servlet-class>
+    <context-param>
+        <param-name>contextConfigLocation</param-name>
+        <param-value>WEB-INF/classes/spring/repository.xml</param-value>
+    </context-param>
 
-    <init-param>
-      <param-name>javax.ws.rs.Application</param-name>
-      <param-value>org.fcrepo.http.commons.FedoraApplication</param-value>
-    </init-param>
-    <load-on-startup>1</load-on-startup>
-  </servlet>
+    <listener>
```

Comment:
Change to: https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-webapp/src/main/webapp/WEB-INF/web.xml#L15-L17